hw2/04-background.js

```diff
@@ -1 +1,39 @@
-// Add your code here
+const timeInterval = 3000;
+
+const body = document.body;
+let identifier = null;
+
+let button = document.getElementsByClassName("btn")[0];
+let timeIntervalInput = document.getElementById("timeInput");
```

Comment:
Your button and timeIntervalInput should be const since they don't get updated anywhere.

Reply:
Updated. Thanks for the feedback!